composer.json

```diff
@@ -5,8 +5,8 @@
   "require": {
     "php": ">=7.1",
     "spryker/kernel": "^3.0.0",
```

Comment:
I guess it would be a good thing to use version that supports new Container usage (3.30.0)